theme/theme.inc

```diff
@@ -401,37 +378,15 @@ function template_process_islandora_newspaper_page(array &$variables) {
       $variables['content'] = NULL;
     }
   }
+  // Fallback to image.
```

Comment:
I  know this was my own suggested merge, but since I just did a copy from above this comment is probably invalid and should be removed or amended to 
```// Fallback to image if no viewer selected and its accessible.``` 

Thoughts?